src/model/utils/messageField.ts

```diff
@@ -0,0 +1,9 @@
+export class MessageField {
+    public field: string;
+    public reason: string;
+
+    constructor(field: string, reason: string) {
```

Comment:
Please change to 

constructor(public field: string, public reason: string) {

Newer syntax for constructor, and easier to maintain

Reply:
this is a syntax in the TS for making a field in the class in constructor.